src/commands/ps/index.js

```diff
@@ -163,10 +164,10 @@ Examples:
     $ heroku ps
     === run: one-off dyno
     run.1: up for 5m: bash
-    
```

Comment:
This actually needs to be there for the markdown formatting

Reply:
Gotcha. My linter did that automatically, so I'll be mindful about that going forward.

Reply:
yeah this is one reason why I'd like to make examples more formal, so we don't have to have the entire markdown format in the string